frontend/src/components/PageAfterAuth.jsx

```
@@ -34,7 +34,7 @@ export default class PageAfterAuth extends React.Component {
                 {
                     this.state.courses
                         .map(course =>
-                            <CourseCard g_course={course.g_course} g_city={course.g_city} onDelete={() => this.deleteUserCourse(course.id)}/>
+                            <CourseCard display_name={course.display_name} city={course.city} onDelete={() => this.deleteUserCourse(course.id)}/>
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)